docs/create-packages/Symbol-Packages-V2.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
I don't know much about this flow. Maybe we should get @shishirx34's and @diverdan92's 👀  on it?